src/test/regress/expected/multi_mx_modifying_xacts.out

```diff
@@ -361,7 +361,7 @@ DROP TRIGGER reject_bad_mx ON labs_mx_1220102;
 CREATE CONSTRAINT TRIGGER reject_bad_mx
 AFTER INSERT ON labs_mx_1220102
 DEFERRABLE INITIALLY DEFERRED
-FOR EACH ROW EXECUTE PROCEDURE reject_bad_mx();
+FOR EACH ROW EXECUTE FUNCTION reject_bad_mx();
```

Comment:
if  `PROCEDURE ` is with  pg 12+, could we make the test outputs `PROCEDURE `  instead of `FUNCTION`? It is to get ready for dropping PG 11

Reply:
it is `FUNCTION` in pg12 afaik